parts/linux/cloud-init/artifacts/cgroup-stats.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -o errexit
+
+EVENTS_LOGGING_DIR=/var/log/azure/Microsoft.Azure.Extensions.CustomScript/events/
+EVENTS_FILE_NAME=$(date +%s%3N)
+CSE_STARTTIME=$(date)
+CSE_STARTTIME_FORMATTED=$(date +"%F %T.%3N")
+CSE_ENDTIME_FORMATTED=$(date +"%F %T.%3N")
+NODEPOOL_NAME=$(curl -s -H Metadata:true --noproxy "*" "http://169.254.169.254/metadata/instance?api-version=2021-02-01" | jq -r '.compute.osProfile.computerName')
+VM_TYPE=$(kubectl describe node $NODEPOOL_NAME --kubeconfig var/lib/kubelet/kubeconfig | grep "beta.kubernetes.io/instance-type=" | cut -d '=' -f 2)
```

Comment:
let's avoid calling k8s api. use IMDS for this one if you want the name. but I don't see it emitted anyway.